### PR TITLE
Add XCM Channels

### PR DESCRIPTION
**Pull request by 0xLucca**
This PR adds the overview of xcm channels and tutorials on how to open these channels:

- Develop > Interoperability > XCM Channels
- Tutorials > Interoperability > XCM Channels > Opening HRMP Channels Between Parachains
- Tutorials > Interoperability > XCM Channels > Opening HRMP Channels with System Parachains

### CODE REVIEW

File: develop/interoperability/xcm-channels.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- **Sudo** - if the parachain has a `sudo` pallet configured, the sudo key holder can use the sudo extrinsic to dispatch the necessary channel opening calls
+- **Governance** - the parachain's governance system, such as a council or OpenGov, can be used to authorize the channel opening calls
+- **Privileged Accounts** - the parachain may have other designated privileged accounts that are allowed to dispatch the HRMP channel opening extrinsics
+
+## Where to Go Next
+
+Explore the following tutorials for detailed, step-by-step guidance on setting up cross-chain communication channels in Polkadot:
+
+- [Opening HRMP Channels Between Parachains](/tutorials/interoperability/xcm-channels/para-to-para.md){target=\_blank}
+- [Opening HRMP Channels with System Parachains](/tutorials/interoperability/xcm-channels/para-to-system.md){target=\_blank}
```
Comment (nhussein11) [inline develop/interoperability/xcm-channels.md:63]:
```suggestion
- [Opening HRMP Channels Between Parachains](/tutorials/interoperability/xcm-channels/para-to-para/){target=\_blank}
- [Opening HRMP Channels with System Parachains](/tutorials/interoperability/xcm-channels/para-to-system/){target=\_blank}
```

File: tutorials/interoperability/xcm-channels/para-to-system.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    2. Choose the **`hrmpChannels`** call
+    3. Click the **+** button to execute the query
+    ![](/images/tutorials/interoperability/xcm-channels/para-to-system/hrmp-para-to-system-3.webp)
+    
+3. Examine the query results. You should see output similar to the following:
+    ```json
+    --8<-- 'code/tutorials/interoperability/xcm-channels/para-to-system/hrmp-query-output.json'
+    ```
+
+
```
Comment (nhussein11) [inline tutorials/interoperability/xcm-channels/para-to-system.md:100]:
```suggestion
```

File: tutorials/interoperability/xcm-channels/para-to-system.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+## Procedure for Establishing HRMP Channel
+
+This guide demonstrates opening an HRMP channel between parachain 2500 and system chain Asset Hub (parachain 1000) on the Rococo Local relay chain.
+
+### Step 1 - Fund Parachain Sovereign Account
+<!-- This content will be moved to a new page because it is used in multiple places -->
+The [sovereign account](https://github.com/polkadot-fellows/xcm-format/blob/10726875bd3016c5e528c85ed6e82415e4b847d7/README.md?plain=1#L50){target=_blank} for parachain 2500 on the relay chain must be funded so it can take care of any XCM transact fees.
+
+Use [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank} UI to connect to the relay chain and transfer funds from your account to the parachain 2500 sovereign account.
+![](/images/tutorials/interoperability/xcm-channels/hrmp-channels-2.webp)
```
Comment (nhussein11) [inline tutorials/interoperability/xcm-channels/para-to-system.md:35]:
```suggestion
Use [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank} UI to connect to the relay chain and transfer funds from your account to the parachain 2500 sovereign account.

![](/images/tutorials/interoperability/xcm-channels/hrmp-channels-2.webp)
```

**Review by nhussein11 — APPROVED** [review]
Please, check out those small details, but besides that LGTM! 🚀

File: tutorials/interoperability/xcm-channels/para-to-para.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+    1. Select the **`hrmp`** pallet
+    2. Choose the **`hrmpAcceptOpenChannel`** extrinsic
+    3. Fill in the parameters:
+        - **`sender`** - parachain ID of the requesting chain (in this case, 2500)
+    4. Copy the encoded call data
+    ![](/images/tutorials/interoperability/xcm-channels/para-to-para/hrmp-para-to-para-5.webp)
+    The encoded call data for accepting a channel with parachain 2500 should be `0x3c01c4090000`
+
+### Step 3 - Crafting and Submitting the XCM Message from the Receiver
```
Comment (eshaben) [inline tutorials/interoperability/xcm-channels/para-to-para.md:147]:
```suggestion
### Crafting and Submitting the XCM Message from the Receiver
```

File: tutorials/interoperability/xcm-channels/para-to-system.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    - A relay chain
+    - A parachain
+    - An Asset Hub system chain
+- A wallet containing enough funds to cover transaction fees on each of the participating chains
+
+## Procedure for Establishing HRMP Channel
+
+This guide demonstrates opening an HRMP channel between parachain 2500 and system chain Asset Hub (parachain 1000) on the Rococo Local relay chain.
+
+### Step 1 - Fund Parachain Sovereign Account
```
Comment (eshaben) [inline tutorials/interoperability/xcm-channels/para-to-system.md:30]:
```suggestion
### Fund Parachain Sovereign Account
```

File: tutorials/interoperability/xcm-channels/para-to-system.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+        - Parachain 2500 would be encoded as `c4090000`
+
+    3. Combine the prefix and parachain ID encoding to form the full sovereign account address:
+
+        The sovereign account of parachain 2500 in relay chain will be `0x70617261c4090000000000000000000000000000000000000000000000000000`
+        and the SS58 format of this address is `5Ec4AhPSY2GEE4VoHUVheqv5wwq2C1HMKa7c9fVJ1WKivX1Y`
+    
+    To perform this conversion, you can also use the **"Para ID" to Address** section in [Substrate Utilities](https://www.shawntabrizi.com/substrate-js-utilities/).
+
+### Step 2 - Create Establish Channel with System Extrinsic
```
Comment (eshaben) [inline tutorials/interoperability/xcm-channels/para-to-system.md:56]:
```suggestion
### Create Establish Channel with System Extrinsic
```

File: tutorials/interoperability/xcm-channels/para-to-system.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+    1. Select the **`hrmp`** pallet
+    2. Choose the **`establish_channel_with_system`** extrinsic
+    3. Fill in the parameters:
+        - **`target_system_chain`** - parachain ID of the target system chain (in this case, 1000)
+    4. Copy the encoded call data
+    ![](/images/tutorials/interoperability/xcm-channels/para-to-system/hrmp-para-to-system-1.webp)
+    The encoded call data for establishing a channel with system parachain 1000 should be `0x3c0ae8030000`
+
+### Step 3 - Crafting and Submitting the XCM Message
```
Comment (eshaben) [inline tutorials/interoperability/xcm-channels/para-to-system.md:72]:
```suggestion
### Crafting and Submitting the XCM Message
```

File: tutorials/interoperability/xcm-channels/para-to-para.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+
+To accept the HRMP channel opening, you need to create and submit an XCM message that includes the encoded `hrmpAcceptOpenChannel` call data from the previous step. This process is similar to the one described in the previous section's [Step 3 - Crafting and Submitting the XCM Message](#step-3---crafting-and-submitting-the-xcm-message-from-the-sender), with a few key differences:
```
Comment (eshaben) [inline tutorials/interoperability/xcm-channels/para-to-para.md:149]:
```suggestion
To accept the HRMP channel opening, you need to create and submit an XCM message that includes the encoded `hrmpAcceptOpenChannel` call data from the previous step. This process is similar to the one described in the previous section, [Crafting and Submitting the XCM Message](#crafting-and-submitting-the-xcm-message-from-the-sender), with a few key differences:
```

File: develop/interoperability/xcm-channels.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+To view the current values of these parameters in the Polkadot network:
+
+1. Visit [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws#/explorer), navigate to the **Developer** dropdown and select the **Chain state** option
+
+    ![](/images/develop/interoperability/xcm-channels/xcm-channels-1.webp)
+
+2. Query the chain configuration parameters. The result will display the current settings for all the Polkadot network parameters, including the HRMP channel settings
+    1. Select **configuration**
+    2. Choose the **activeConfig()** call
```
Comment (eshaben) [inline develop/interoperability/xcm-channels.md:40]:
```suggestion
    1. Select **`configuration`**
    2. Choose the **`activeConfig()`** call
```

**Review by eshaben — APPROVED** [review]
Nice! Looks good!

**Pull request merged**